src/main.js

```diff
@@ -164,10 +166,16 @@ module.exports = {
               }
             }
             let range
+            const msgLine = line - 1
+            const msgCol = column ? column - 1 : column
```

Comment:
This works fine, but it might be nice to be a little more explicit with something like:

```js
const msgCol = column > 0 ? column - 1 : 0
```

or even better (I think):

```js
const msgCol = Math.max(0, column - 1)
```